ledger-service/http-json/src/itlib/scala/http/AbstractWebsocketServiceIntegrationTest.scala

```diff
@@ -1147,39 +1154,46 @@ abstract class AbstractWebsocketServiceIntegrationTest(val integration: String)
 
   // Following #16782, we use canton community edition over sandbox-on-x.
   // Pruning is only enabled on enterprise edition, TODO: #16832
-  "fail reading from a pruned offset" ignore withHttpService { fixture =>
-    import fixture.{uri, client}
-    for {
-      aliceH <- fixture.getUniquePartyAndAuthHeaders("Alice")
-      (alice, aliceHeaders) = aliceH
-      offsets <- offsetBeforeAfterArchival(alice, fixture, aliceHeaders)
-      (offsetBeforeArchive, offsetAfterArchive) = offsets
-
-      pruned <- PruneGrpc.ParticipantPruningServiceGrpc
-        .stub(client.channel)
-        .prune(
-          PruneGrpc.PruneRequest(
-            pruneUpTo = domain.Offset unwrap offsetAfterArchive,
-            pruneAllDivulgedContracts = true,
-          )
-        )
-      _ = pruned should ===(PruneGrpc.PruneResponse())
-
-      // now query again with a pruned offset
-      jwt <- jwtForParties(uri)(List(alice), List(), testId)
-      query = s"""[{"templateIds": ["Iou:Iou"]}]"""
-      streamError <- singleClientQueryStream(jwt, uri, query, Some(offsetBeforeArchive))
-        .runWith(Sink.seq)
-        .failed
-    } yield inside(streamError) { case t: PeerClosedConnectionException =>
-      // TODO #13506 descriptive/structured error.  The logs when running this
-      // test include
-      //     Websocket handler failed with FAILED_PRECONDITION: PARTICIPANT_PRUNED_DATA_ACCESSED(9,0):
-      //     Transactions request from 0000000000000006 to 0000000000000008
-      //     precedes pruned offset 0000000000000007
-      // but this doesn't propagate to the client
-      t.closeCode should ===(1011) // see RFC 6455
-      t.closeReason should ===("internal error")
+  "fail reading from a pruned offset" ifSupportsPruning_- {
```

Comment:
We replace `ignore` with `ifSupportsPrunning_-`, so it will be run in the EE case.